src/Sulu/Bundle/ReferenceBundle/Infrastructure/Doctrine/Subscriber/DoctrineReferenceSubscriber.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ReferenceBundle\Infrastructure\Doctrine\Subscriber;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\PreFlushEventArgs;
+use Doctrine\ORM\Events;
+use Sulu\Bundle\ReferenceBundle\Domain\Model\ReferenceInterface;
+use Sulu\Bundle\ReferenceBundle\Domain\Repository\ReferenceRepositoryInterface;
+
+class DoctrineReferenceSubscriber implements EventSubscriber
```

Comment:
This can be removed as handled in the collector but will be handled in future in the event subscriber and command